apps/storefront/src/pages/PDP/index.test.tsx

```diff
@@ -0,0 +1,568 @@
+      edges: [
+        {
+          node: shoppingList,
+        },
+      ],
+    };
+
+    const productsResponse = {
+      productsSearch: [
+        {
+          id: 114,
+          name: 'Product with variant',
+          sku: 'test-1',
+          variants: [
+            {
+              variant_id: 179,
+              product_id: 114,
+              sku: 'test-1-1',
+              option_values: [
+                {
+                  id: 103,
+                  label: 'Silver',
+                  option_id: 114,
+                  option_display_name: 'Color',
+                },
+              ],
+            },
+            {
+              variant_id: 180,
+              product_id: 114,
+              sku: 'test-1-2',
+              option_values: [
+                {
+                  id: 104,
+                  label: 'Black',
+                  option_id: 114,
+                  option_display_name: 'Color',
+                },
+              ],
+            },
+          ],
+          imageUrl: '',
+          modifiers: [],
+          options: [
+            {
+              option_id: 114,
+              display_name: 'Color',
+              sort_order: 0,
+              is_required: true,
+            },
+          ],
+          optionsV3: [
+            {
+              id: 114,
+              product_id: 114,
+              name: 'Color1742885038-114',
+              display_name: 'Color',
+              option_values: [
+                {
+                  id: 103,
+                  label: 'Silver',
+                  sort_order: 0,
+                  is_default: false,
+                },
+                {
+                  id: 104,
+                  label: 'Black',
+                  sort_order: 1,
+                  is_default: false,
+                },
+              ],
+              config: [],
+            },
+          ],
+          channelId: [],
+          productUrl: '',
+        },
+      ],
+    };
+
+    vi.spyOn(graphqlModule, 'getB2BShoppingList').mockResolvedValue(shoppingListResponse);
+    vi.spyOn(graphqlModule, 'searchB2BProducts').mockResolvedValue(productsResponse);
+    vi.spyOn(graphqlModule, 'addProductToShoppingList').mockRejectedValue(
+      new Error('Unexpected error'),
+    );
+
+    vi.mock('@/utils/b3Tip', () => ({
+      globalSnackbar: {
+        error: vi.fn(),
+        success: vi.fn(),
+      },
+    }));
+
+    renderWithProviders(<PDP />, {
+      preloadedState: {
+        global: globalState,
+        company: companyStateWithSuperAdminUser,
+      },
+    });
+
+    await waitForElementToBeRemoved(() => screen.queryByText(/loading/i));
+
+    expect(screen.getByText('Add to shopping list')).toBeInTheDocument();
+
+    await userEvent.click(screen.getByText('Test Shopping List 1'));
+    await userEvent.click(screen.getByText('OK'));
+
+    expect(globalSnackbar.error).toHaveBeenCalledWith(
+      'Something went wrong. Please try again.',
+      expect.anything(),
```

Comment:
[Suggestion]
It seems the second argument is always the same, maybe we can replace with what is the expected value
```suggestion
      {
        isClose: true,
      },
```

Reply:
updated